### ISSUE TITLE
Test jupyterlab-plotly, jupyterlab-dash and jupyter-matplotlib

### ISSUE THREAD

**Issue by ca-scribner** [StatCan/kubeflow-containers#104]
Do they work?  jupyter-matplotlib gave me these errors:

```
matplotlib notebook
matplotlib.pyplot as plt
plt.plot([[0,0], [1,1]], label='line')
plt.show()
```

Make sure they do, and while doing that build demo notebooks that can be added to jupyter-notebooks repo so we have both test cases and demos for users

**Comment by ca-scribner** [comment]
the matplotlib extension is also installed twice (once by us, and once upstream).  Remove the second one?

**Comment by ca-scribner** [comment]
jupyterlab-dash has been deprecated in place of jupyter-dash.  Update to the newer one

**Comment by Ito-Matsuda** [comment]
Confirming that I get the "IPython is not defined" with `%matplotlib notebook`
From what I understand we should use `%matplotlib widget` instead. See [matplotlib jupyter-notebooks](https://matplotlib.org/users/interactive.html#jupyter-notebooks-lab) or the [matplotlib ipympl github proj](https://github.com/matplotlib/ipympl)

So your snippet of code became 
```
%matplotlib widget 
import matplotlib.pyplot as plt
import numpy as np
plt.plot([[0,0], [1,1]], label='line')
plt.show()
```

Removed the extra installation of `jupyter-matplotlib` in the minimal-notebook cpu and tested in the Python 3 notebook successfully. Graph was interact-able  (moused over graph and X+Y co-ords appeared below as opposed to a normal graph created by just running the code sans `%matplotlib widget`)
Saved+downloaded the notebook containing the code above and the resulting graph. 


Still TODO:
- Update jupyterlab-dash to jupyter-dash.
- Test jupyter-dash 
- Test jupyter-plotly

**Comment by Ito-Matsuda** [comment]
**jupyter-plotly** works with some examples from [plotly.com basic charts](https://plotly.com/python/basic-charts/) the following is from bar charts
```
import plotly.express as px
data_canada = px.data.gapminder().query("country == 'Canada'")
fig = px.bar(data_canada, x='year', y='pop')
fig.show()
```

**jupyterlab-dash**
Example(s) from the jupyterlab-dash [git page](https://github.com/plotly/jupyterlab-dash) do not produce expected output.
Does not appear in installed extensions.

<br>

**TLDR;**
jupyter-matplotlib: works, removed the duplicate installation still worked, have the ipynb file.
jupyter-plotly: works, have the ipynb file. 
jupyterlab-dash: no bueno
